src/snowflake/cli/plugins/nativeapp/version/commands.py

```diff
@@ -137,6 +150,12 @@ def drop(
     Drops a version defined in your application package. Versions can either be passed in as an argument to the command or read from the `manifest.yml` file.
     Dropping patches is not allowed.
     """
+
+    if cli_context.project_definition.native_app is None:
+        raise NoProjectDefinitionError(
+            project_type="native_app", project_file=cli_context.project_root
+        )
```

Comment:
How about introducing a helper
`assert_project_type(project_type="native_app")`

this would reduce the code duplication. WDYT?

Reply:
updated